website/docs/index.html.markdown

```diff
@@ -89,6 +90,17 @@ Users of the provider before version `2.x` will need to make a few adjustments t
 
 Please see the [latest provider configuration docs](/docs/providers/newrelic/guides/provider_configuration.html) for the current recommended configuration settings.
 
+## Support for 1.x
```

Comment:
This seems like it deserves a (highly visible) statement at the beginning of any top-level 1.x documentation files as well.

Reply:
Agreed.  Top of the page for each of the v1?  Are we good with the words before copying?